src/Views/SuccessView.vala

```diff
@@ -31,7 +31,12 @@ public class SuccessView : AbstractInstallerView {
         primary_label.get_style_context ().add_class (Granite.STYLE_CLASS_H2_LABEL);
 
         var secondary_label = new Gtk.Label (
-            _("Your device will automatically restart to %s in %i seconds.").printf (Utils.get_pretty_name (), RESTART_TIMEOUT) + " " +
+            _("%s was successfully installed.").printf (Utils.get_pretty_name ()) + " " +
```

Comment:
Looks like we use “has been installed” for apps in AppCenter and Sideload, so I would match that here. Plus we don’t need to specify “successfully” when it is implied.


```suggestion
            _("%s has been installed.").printf (Utils.get_pretty_name ()) + " " +
```